.github/workflows/main_win.yaml

```diff
@@ -133,6 +134,7 @@ jobs:
     permissions:
       packages: write
     runs-on: windows-2019
+    timeout-minutes: 60
```

Comment:
Are we sure 60 mins is enough to build Windows container?  We build two containers here.

Reply:
Unsure, i don't see any recent builds in Datadog for it. Made #3416 to test but i guess i can bump this to 2h for now.

Reply:
Made #3417 as well and will re-run it couple times. Bumped it to 2h for now for extra safety.

Reply:
I got "only" 35 minutes or so, for what it's worth